### DISCUSSION TITLE
Running Balance Column

### DISCUSSION THREAD

**Discussion by s38b35M5** [firefly-iii/firefly-iii#9061] [Firefly III]
Perhaps this is a feature request, but first, I'm wondering if there's a way to add a running balance column so each row/transaction shows the balance as affected by the transaction?

This would be very useful in account view and perhaps other views, if it isn't available presently.

**Comment by JC5** [comment]
There is already a ticket for this, yes :)

**Reply by s38b35M5** [reply]
Ah! Apologies! I am not seeing it. Sorry for the dup!

**Reply by s38b35M5** [reply]
@JC5 can you point me to it when you get a chance? Thanks!

**Reply by Tejeev** [reply]
I found one but it was closed.  Could someone point to the current one?

**Comment by Tejeev** [comment]
I'm really struggling.  I don't understand why all the apps appear to have done away with running balances..  To me, it's such a core basic feature of any statement or budgeting tool, that it is the feature on which my decision to use or not use a tool pivots.
